### DISCUSSION TITLE
Disabled cameras visible in Review pane

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#15755] [General]
FYI: Frigate 15, Beta 4
I have 5 cameras defined in my frigate yml configuration, two of which are disabled. In Camera view only the 3 enabled cameras are visible, as expected, however when switching to Review all 5 cameras appear. 
Not sure if this is intentional or a bug.

81Xats
![frigate](https://github.com/user-attachments/assets/d85c1c05-ee80-4b04-a001-0c2865235dc5)

**Comment by NickM-27** [comment]
this is intentional, just because a camera is disabled now doesn't mean it was disabled at the point in time that you are currently scrubbing

**Reply by 18923** [reply]
Makes sense...
Happy New Year to the team.

**Comment by 18923** [comment]
.
